src/cljs/rems/table2.cljs

```diff
@@ -0,0 +1,134 @@
+(ns rems.table2
```

Comment:
I guess the plan is to replace rems.table with this ns?

Reply:
Yes